src/pages/products/SingleProduct.jsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from 'react'
+import { Navbar } from 'components/navbar/Navbar.jsx'
```

Comment:
you can use an index,js in components and export all from the index and import at once rather than individually importing them, if you have an index, then you should use that.